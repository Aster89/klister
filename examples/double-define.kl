#lang kernel

(import "define-syntax-rule.kl")

(define-syntax-rule (double-define x y value)
  (group
    (define x value)
    (define y value)))

(double-define a b 1)
(example a)
(example b)
