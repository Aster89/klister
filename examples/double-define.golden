Signal 1 : Signal
Signal 1 : Signal
